### PR TITLE
Completed css-positioning

### PR DESCRIPTION
**Pull request by amorabryan**
image
image
image

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  bottom: 0;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
```
Comment (kurtyazdizadeh) [inline css-positioning/absolute.css:33]: overlay should also have a `position` set

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
One small change, just missing some positioning on the overlay in the `absolute.css` stylesheet

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Comment by amorabryan** [comment]
image

**Pull request merged**